### ISSUE TITLE
catwalk deletion issue

### ISSUE THREAD

**Issue by rrusciguy** [TeamDmfMM/Catwalks-4#12]
i was experimenting with them in creative, and if I break a catwalk and place a new one down in its place, the collision and the graphics don't match up, although a blowtorch does successfully swap between rail and no rail and such the graphics don't update along with it while the collision does. I've even placed other blocks in their place, to see if that would overwrite it somehow, but the issue persists

**Comment by dmf444** [comment]
I'll have to check it out, probably something to do with our cacheing

**Comment by gazpachoking** [comment]
Yeah, I'm having the same issue. If I break and replace a catwalk, it will render forever in the same state as it was broken, even if you toggle the sides later with the blowtorch.

**Comment by OhiraKyou** [comment]
Is this with OptiFine installed? I noticed the same thing the first time I tried the mod. Setting the OptiFine support config variable to true fixed that for me, but it then lead to a consistent crash (issue #10)

**Comment by rrusciguy** [comment]
I do not have optifine installed, no

**Comment by SeriousCreeper** [comment]
This is still an issue and makes this pretty unusable if you want to replace any of your modified catwalks. 

Video of it happening: https://streamable.com/kwew2i

Also placing catwalks against others is really finnicky, i keep trying to place them in a straight line and either they are placed on the side or not at all until i find the right spot to click
